packages/lesswrong/components/review/NominationsPage.tsx

```diff
@@ -73,7 +73,7 @@ const dateStr = (startDate?: Date) =>
 export const allPostsParams = (reviewYear: ReviewYear=REVIEW_YEAR) => {
   const startDate = getReviewPeriodStart(reviewYear).toDate()
   const endDate = getReviewPeriodEnd(reviewYear).toDate()
-  return {after: dateStr(startDate), before: dateStr(endDate), timeframe: 'yearly', frontpage: 'true', unnominated: 'true'}
+  return {after: dateStr(startDate), before: dateStr(endDate), sortedBy: 'top', timeframe: 'yearly', frontpage: 'true', unnominated: 'true', limit: "100"}
```

Comment:
Looks like this gets turned into a URL query string by concatenation? so probably correct that this is a string